sources/packages/web/src/views/student/StudentProfile.vue

```diff
@@ -2,13 +2,31 @@
   <student-page-container>
     <template #header>
       <header-navigator v-if="editMode" title="Student" subTitle="Profile" />
+      <v-row v-else class="text-center">
```

Comment:
Why didn't we re-use the same header-navigator on create mode?

Reply:
This is the only place (so far) where there is a "centered" header in a different way from the other places.
I did not create a component for this and it does not look like our usual header.

![image](https://user-images.githubusercontent.com/61259237/182950272-e9cdedf9-c200-4069-aaa2-c45eaeb090f0.png)

Reply:
Makes sense. If this component repeats, then we can think about adding a slot to handle this in full page container.